### PR TITLE
Implement BoxAirMode flag in MSP_STATUS

### PR DESCRIPTION
**Pull request by gaelj**
Fixes issue  #2156

### CODE REVIEW

File: src/main/config/runtime_config.h [hunk 1]
```diff
@@ -56,7 +56,8 @@ extern uint16_t flightModeFlags;
 // It is much more memory efficient than full map (uint32_t -> uint8_t)
 #define FLIGHT_MODE_BOXID_MAP_INITIALIZER {                             \
         BOXANGLE, BOXHORIZON, BOXMAG, BOXBARO, BOXGPSHOME, BOXGPSHOLD,  \
-        BOXHEADFREE, -1, BOXPASSTHRU, BOXSONAR, BOXFAILSAFE, BOXGTUNE}  \
+        BOXHEADFREE, -1, BOXPASSTHRU, BOXSONAR, BOXFAILSAFE, BOXGTUNE,  \
```
Comment (ledvinap) [inline src/main/config/runtime_config.h]: Only flightModeFlags_e -> BOXID goes here. One entry for each flightModeFlags_e value.

File: src/main/io/msp.c [hunk 2]
```diff
@@ -341,6 +341,8 @@ static uint32_t packFlightModeFlags(void)
             continue;                 // this flightmode is not copied
         if(FLIGHT_MODE(1 << i))
             boxEnabledMask |= 1 << flightMode_boxId_map[i];
+        if (rcModeIsActive(BOXAIRMODE) && flightMode_boxId_map[i] == BOXAIRMODE)
+            boxEnabledMask |= 1 << flightMode_boxId_map[i];
```
Comment (ledvinap) [inline src/main/io/msp.c]: Just add `MB(BOXAIRMODE)` to `rcModeCopyMask`. That's all it takes to pack new flag ..

**Comment by gaelj** [comment]
@hydra, I believe this PR should be merged in 1.13 RC2.

File: src/main/io/msp.c (hunk 2 above)
Comment (gaelj) [inline src/main/io/msp.c]: Indeed :)

**Comment by gaelj** [comment]
Thanks @ledvinap. Now it's nice and clean.

**Comment by hydra** [comment]
@gaelj can you squash the commits and force push this please?  I will merge once you do it.

**Comment by gaelj** [comment]
@hydra : squash done

**Comment by hydra** [comment]
@gaelj thanks, merging.

**Pull request merged**